laterpay/laterpay.php

```diff
@@ -4,7 +4,7 @@
  * Plugin URI: https://github.com/laterpay/laterpay-wordpress-plugin
  * Description: Sell digital content with LaterPay. It allows super easy and fast payments from as little as 5 cent up to 149.99 Euro at a 15% fee and no fixed costs.
  * Author: LaterPay GmbH
- * Version: 2.7.0
+ * Version: 2.8.0
```

Comment:
We will update version number in a separate PR, see e.g https://github.com/laterpay/laterpay-wordpress-plugin/pull/1322, remove this change for now.

Reply:
Done